tests/unit/mainwindow/test_prompt.py

```diff
@@ -82,7 +82,9 @@ def test_backspacing_path(self, qtbot, tmp_path, get_prompt):
                 qtbot.keyPress(prompt._lineedit, Qt.Key_Backspace)
 
         # foo should get completed from f
+
         prompt.item_focus('next')
+
         assert prompt._lineedit.text() == str(testdir / 'foo')
```

Comment:
```suggestion
        # .. and foo should get completed from f
        prompt.item_focus('next')
        assert prompt._lineedit.text() == str(tmp_path)
        prompt.item_focus('next')
        assert prompt._lineedit.text() == str(testdir / 'foo')
```

(untested). Also I'd prefer removing the blank lines you've added above.